---
layout: post
title: Readings
cover: readings.jpg
categories: posts
permalink: readings
---

Below are several papers which you may find useful for the rotation or are a fun read (suggested by faculty).

## Statistical Physics and Biology
* [**Theory in Biology: Figure 1 or Figure 7?**](http://rpdata.caltech.edu/publications/phillips2015.pdf) by Rob Phillips. Cell Press (2015) \| A disscussion about the role of the theoretical model in an experimental paper (figure 1 vs figure 7).

* [**Models in biology: 'accurate descriptions of our pathetic thinking'**](https://bmcbiol.biomedcentral.com/track/pdf/10.1186/1741-7007-12-29) Jeremy Gunawardena.  BMC Biology (2014) \| This paper discusses the ideas for thinking about models in biology. It will breakdown a couple quantitiative models to their logical structure (assumptions and conclusions) to show a guideline for prospective model builders.

* [**A First Exposure to Statistical Mechanics for Life Scientists**](http://www.rpdata.caltech.edu/publications/Garcia2007b.pdf) by Hernan G. Garcia, Jané Kondev, Julie A. Theriot, and Rob Phillips. Rejected by Am. J. Phys, 2007. \| This paper summarizes the basics of statistical mechanics and presents them in an array of biological contexts.

* [**Transcriptional Regulation by the Numbers 1: Models**](http://www.rpdata.caltech.edu/publications/bintu2005a.pdf) by Lacramiaora Bintu, Nicolas E. Buchler, Hernan G. Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, and Rob Phillips. Curr. Op. Gen. Dev. (2005) \| This work shows how thermodynamic models are used to describe the behavior of a variety of regulatory architectures.

* [**Transcriptional Regulation by the Numbers 2: Applications**](http://www.rpdata.caltech.edu/publications/bintu2005b.pdf) by Lacramiaora Bintu, Nicolas E. Buchler, Hernan G. Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, Thomas Kuhlman, and Rob Phillips. Curr. Op. Gen. Dev. (2005) \| Here, our discussion of thermodynamic models of genetic regulation are expanded to examine how these input/output functions can be used to make predictions regarding transcriptional outputs.

* [**Cell Biology by the Numbers**](http://book.bionumbers.org) by Ron Milo and Rob Phillips. Garland Science (2015) \| This is a very useful resources that gives numbers for various biological miscellany.

* [**Measurement of the Force-Velocity Relation for Growing Microtubules**](http://science.sciencemag.org/content/278/5339/856) by Marileen Dogterom, Bernard Yurke. Science (1997) \| Here is a beautiful microtubule experiment.

* [**Quantitative dissection of the simple repression input-output function**](http://www.rpdata.caltech.edu/publications/Garcia2011c.pdf) by Hernan G. Garcia and Rob Phillips. PNAS (2011). \| This is the paper discusses the transcriptional model discussed in class, which is compared to the data collected by Hernan.


## Interesting Methods

* [**Gene Regulation at the Single Cell  Level**](http://rpdata.caltech.edu/course_papers/Rosenfeld2005.pdf) by Nitzan Rosenfeld, Jonathan W. Young, Uri Alon, Peter S. Swain, and Michael B. Elowitz. Science (2005) \| This was the first publication of the method of counting proteins by the dilution method. This is the method we will use for counting the number of repressors under varying growth conditions.

## Relevant Project Literature

* [**Quantitative Dissection of the Simple Repression Input-Output Function**](http://www.rpdata.caltech.edu/publications/Garcia2011c.pdf) by Hernan G. Garcia and Rob Phillips. PNAS (2011). \| This work laid the foundation upon which all of our studies regarding the fold-change in gene expression
